src/utils/xhr-utils.js

```diff
@@ -421,3 +421,32 @@ export function getViewerAuthTokenIfAvailable(element) {
   // If crossorigin attribute is missing, always resolve with undefined.
   return Promise.resolve(undefined);
 }
+
+/**
+ * Format data for pingback.
+ * @param {any} obj
+ * @return {string}
+ */
+export function stringifyForPingback(obj) {
```

Comment:
That should go into some file inside the `amp-subscriptions` extension. E.g. `amp-subscriptions/0.1/stringifyForPingback.js`.